peps/pep-0772.rst

```diff
@@ -0,0 +1,1612 @@
+PEP: 772
+Title: A Python Installation Manager for Windows
+Author: Steve Dower
+PEP-Delegate: TBD
+Discussions-To: https://discuss.python.org/t/pep-772-a-python-installation-manager-for-windows/77900
+Status: Draft
+Type: Standards Track
+Topic: Release
+Created: 21-Jan-2025
+Post-History: 18-Dec-2024
+
+
+Abstract
+========
+
+Installation of the python.org Python distribution on Windows is complex.
+There are three main approaches with roughly equivalent levels of user
+experience, and yet all of these suffer from different limitations, including
+failing to satisfy modern usage scenarios. This PEP proposes a design for
+a single Windows install workflow tool that satisfies all the needs of the
+existing installers for the platform, while avoiding most of their limitations,
+and provides the core team with the ability to manage releases for many years
+to come.
+
+Reader's Note
+-------------
+
+This document is a detailed *proposal*, intended to assist in an "approve" or
+"reject" decision on the replacement of the existing install formats for Python
+on Windows with a new format. It is not intended to be a binding specification,
+nor is it user documentation or an interoperability specification. The actual
```

Comment:
As such, should this be `Type: Informational` rather than `Type: Standards Track`?

Reply:
Open for discussion, but generally "Informational" PEPs never reach "Final", and this one should. Plus it's somewhat tied to a release, even though it's not tied to the language.